### ISSUE TITLE
Checkpoint 3 Ready

### ISSUE THREAD

**Issue by Belloche** [Belloche/DraftShare#9]
@pawaitemadisoncollege Checkpoint 3 is ready for review. Gotta say the worst thing working on this was the scourge of Daylight Savings Time. I legitimately didn't think I would have a real reason to hate it other than just waking up an hour early and being confused for a couple weeks but thanks to the power of Java, I now hate DST at full potential.
Right now the only site that I have that uses the authentication is when you click on View Draft in one of the thumbnails on the main page. The user I have to log in is pawaite with the password d1556uy
The view draft page is also a template currently but I am working on getting it fully implemented. I'm also curious about something else, regarding logging in before you click on view draft. I want to get a user logged in potentially before they click on view draft, you can see a Log In button in the top right of the nav, and I want it to go away once you're logged in. How could I accomplish that?

**Comment by pawaitemadisoncollege** [comment]
Ha ha! I'm with you on DST @Belloche!

**Can you bounce your ec2 instance? I'm to able to access it this morning and I'm assuming it just needs a restart.**


- [ ] As you add additional resources, be sure to also add them to the list of restricted items as needed. https://github.com/Belloche/DraftShare/blob/2cad44ea371e471ecde1f8ee73572d27c151749a/src/main/webapp/WEB-INF/web.xml#L20-L28. 
- [ ] Reminder to implement the GenericDao and remove the entity-specific daos.

**Comment by Belloche** [comment]
@pawaitemadisoncollege I've rebooted the instance and it seems to be working as expected now. I've been seeing a lot of issues with my instance going down a lot, is that mostly because of deploying and the system taking time to get used to the new code?

**Comment by pawaitemadisoncollege** [comment]
@Belloche Yes! the site is up now, but I am getting the following the pawaite/d1556uy uid/pwd combo:

Is there another uid/pwd that you can confirm is working out there?
![Screen Shot 2021-03-24 at 12 27 08 PM](https://user-images.githubusercontent.com/13874466/112356246-6576bf80-8c9c-11eb-9eff-a10111345b0a.png)

**Comment by pawaitemadisoncollege** [comment]
RE: 

> I've been seeing a lot of issues with my instance going down a lot, 

Do you have the automatic reboot set up when a status check fails?  There's been a fair amount of discussion about this and ideas on ways to troubleshoot the root cause in slack? Have you tried some of the suggestions there?

**Comment by pawaitemadisoncollege** [comment]
@Belloche - tried with the uid/pwd pwaite/d1556uy which results in: 
<img width="1788" alt="Screen Shot 2021-03-27 at 9 33 40 AM" src="https://user-images.githubusercontent.com/13874466/112724118-aa3e6880-8edf-11eb-8de9-0784eee11296.png">


Does login actually work for you on the aws-deployed app? Seems like maybe an issue with pathing if there's a 400?

**Comment by pawaitemadisoncollege** [comment]
@Belloche UPDATE: I see now, the login button does not work, but the view full draft link does. All good there! CP 3 is set.
